proto/proto/penumbra/core/transaction/v1alpha1/transaction.proto

```diff
@@ -73,6 +73,54 @@ message Action {
   }
 }
 
+message TransactionView {
+  // A list views into of actions (state changes) performed by this transaction.
+  repeated ActionView action_views = 1;
+  // The maximum height that this transaction can be included in the chain.
+  //
+  // If zero, there is no maximum.
+  uint64 expiry_height = 2;
+  // The chain this transaction is intended for.  Including this prevents
+  // replaying a transaction on one chain onto a different chain.
+  string chain_id = 3;
+  // The transaction fee.
+  crypto.v1alpha1.Fee fee = 4;
+  // A list of clues for use with Fuzzy Message Detection.
+  repeated crypto.v1alpha1.Clue fmd_clues = 5;
+  // An optional plaintext memo. It will only be populated if there are
+  // outputs in the actions of this transaction. 528 bytes.
+  optional bytes plaintext_memo = 6;
+}
+
+// A view of a specific state change action performed by a transaction.
+message ActionView {
+  oneof action {
+    Spend spend = 1;
+    Output output = 2;
```

Comment:
These ones (and the `Swap`, `SwapClaim` below) are the four that need replacing with view-specific protos, right?